modules/ROOT/pages/fd-connection-view-audit-logs.adoc

```diff
@@ -0,0 +1,26 @@
+= View Audit Logs for Connections That are Shared
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.
+
+To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:
+
+* The time and date of the action
+* The product in which the action was taken (i.e., Flow Designer)
```

Comment:
```suggestion
* The product in which the action was taken (for example, Flow Designer)
```

Reply:
"i.e." means "that is to say." It doesn't mean "for example."